[2021-07-19 19:40:43,662] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: test.training_model_A 2021-07-18T00:00:00+00:00 [queued]>
[2021-07-19 19:40:43,668] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: test.training_model_A 2021-07-18T00:00:00+00:00 [queued]>
[2021-07-19 19:40:43,668] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-07-19 19:40:43,668] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2021-07-19 19:40:43,668] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-07-19 19:40:43,683] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): training_model_A> on 2021-07-18T00:00:00+00:00
[2021-07-19 19:40:43,684] {standard_task_runner.py:52} INFO - Started process 13514 to run task
[2021-07-19 19:40:43,687] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'test', 'training_model_A', '2021-07-18T00:00:00+00:00', '--job-id', '6', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/example.py', '--cfg-path', '/tmp/tmpo3po8hd5', '--error-file', '/tmp/tmpyyymeeui']
[2021-07-19 19:40:43,687] {standard_task_runner.py:77} INFO - Job 6: Subtask training_model_A
[2021-07-19 19:40:43,712] {logging_mixin.py:104} INFO - Running <TaskInstance: test.training_model_A 2021-07-18T00:00:00+00:00 [running]> on host DESKTOP-4B4VMOV.localdomain
[2021-07-19 19:40:43,751] {taskinstance.py:1300} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=test
AIRFLOW_CTX_TASK_ID=training_model_A
AIRFLOW_CTX_EXECUTION_DATE=2021-07-18T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-07-18T00:00:00+00:00
[2021-07-19 19:40:43,752] {python.py:151} INFO - Done. Returned value was: 9
[2021-07-19 19:40:43,769] {taskinstance.py:1204} INFO - Marking task as SUCCESS. dag_id=test, task_id=training_model_A, execution_date=20210718T000000, start_date=20210719T174043, end_date=20210719T174043
[2021-07-19 19:40:43,793] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-07-19 19:40:43,818] {local_task_job.py:149} INFO - Task exited with return code 0
[2021-07-19 23:18:24,985] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: test.training_model_A 2021-07-18T00:00:00+00:00 [queued]>
[2021-07-19 23:18:24,993] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: test.training_model_A 2021-07-18T00:00:00+00:00 [queued]>
[2021-07-19 23:18:24,993] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-07-19 23:18:24,993] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2021-07-19 23:18:24,993] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-07-19 23:18:25,017] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): training_model_A> on 2021-07-18T00:00:00+00:00
[2021-07-19 23:18:25,019] {standard_task_runner.py:52} INFO - Started process 27536 to run task
[2021-07-19 23:18:25,021] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'test', 'training_model_A', '2021-07-18T00:00:00+00:00', '--job-id', '41', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/example.py', '--cfg-path', '/tmp/tmp3zqautnp', '--error-file', '/tmp/tmpes28yyde']
[2021-07-19 23:18:25,021] {standard_task_runner.py:77} INFO - Job 41: Subtask training_model_A
[2021-07-19 23:18:25,054] {logging_mixin.py:104} INFO - Running <TaskInstance: test.training_model_A 2021-07-18T00:00:00+00:00 [running]> on host DESKTOP-4B4VMOV.localdomain
[2021-07-19 23:18:25,110] {taskinstance.py:1300} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=test
AIRFLOW_CTX_TASK_ID=training_model_A
AIRFLOW_CTX_EXECUTION_DATE=2021-07-18T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-07-18T00:00:00+00:00
[2021-07-19 23:18:25,111] {python.py:151} INFO - Done. Returned value was: 7
[2021-07-19 23:18:25,150] {taskinstance.py:1204} INFO - Marking task as SUCCESS. dag_id=test, task_id=training_model_A, execution_date=20210718T000000, start_date=20210719T211824, end_date=20210719T211825
[2021-07-19 23:18:25,180] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-07-19 23:18:25,192] {local_task_job.py:149} INFO - Task exited with return code 0
